networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -13,7 +13,7 @@ The Kubernetes NMState Operator provides a Kubernetes API for performing state-d
 Red Hat supports the Kubernetes NMState Operator in production environments on bare metal platform installations only. For other platforms, the Kubernetes NMState Operator is a Technology Preview. See the following Technology Preview statement for support limitations when using Kubernetes NMState on platforms other than bare metal.
 ====
 
-include::modules/technology-preview.adoc[]
+include::snippets/technology-preview.adoc[]
```

Comment:
@johnwilkins Can you please review to make sure this is right? Pretty sure it is, but it looks like your update here reverted it from snippets to modules: https://github.com/openshift/openshift-docs/commit/710912baa177078b05d5de072100403398468b8b